main.go

```diff
@@ -69,13 +71,29 @@ func main() {
 	}
 	defer db.Close()
 
-	// var queueBackend queue.QueueBackend
-	// var storageBackend filestore.StorageBackend
+	var queueBackend queue.QueueBackend
+	switch config.Transport.Queue {
+	case "memory":
+		queueBackend = memoryqueue.NewQueue()
+	case "sqs":
```

Comment:
Remove this for now, we'll add it back in when we actually have an SQS queue.

Reply:
Done.